kview/src/main/java/dev/ikm/komet/kview/events/pattern/MakePatternWindowEvent.java

```diff
@@ -0,0 +1,38 @@
+package dev.ikm.komet.kview.events.pattern;
+
+import dev.ikm.komet.framework.events.Evt;
+import dev.ikm.komet.framework.events.EvtType;
+import dev.ikm.komet.framework.view.ViewProperties;
+import dev.ikm.tinkar.terms.EntityFacade;
+
+public class MakePatternWindowEvent extends Evt {
+
+    public static final EvtType<MakePatternWindowEvent> OPEN_PATTERN = new EvtType<>(Evt.ANY, "OPEN_PATTERN");
```

Comment:
Observation & question: Should we have a refactor ticket to create something like a LaunchWindowEvent, with different kinds of windows such as Concept, Pattern, General Editor? 

My concern is a proliferation of events with only one event type. It's good practice to see events holistically.